### DISCUSSION TITLE
sidebars 和docs文件夹如何根据接口返回动态生成目录

### DISCUSSION THREAD

**Discussion by wqinglin** [facebook/docusaurus#9670] [Ideas]
sidebars 和docs文件夹如何根据接口返回动态生成目录

**Comment by OzakIOne** [comment]
Hello,

Could you provide additional information in English regarding your request?
It's better for everyone to keep conversation in English if possible

**Comment by Josh-Cena** [comment]
You may want to read https://github.com/facebook/docusaurus/issues/4138#issuecomment-771042127
